docs/csharp/programming-guide/events/how-to-implement-interface-events.md

```diff
@@ -9,47 +9,47 @@ ms.assetid: 63527447-9535-4880-8e95-35e2075827df
+Declare the event in your class and then invoke it in the appropriate areas.  
   
-    ```csharp
-    namespace ImplementInterfaceEvents  
+```csharp
+namespace ImplementInterfaceEvents  
+{  
+    public interface IDrawingObject  
     {  
-        public interface IDrawingObject  
-        {  
-            event EventHandler ShapeChanged;  
-        }  
-        public class MyEventArgs : EventArgs   
+        event EventHandler ShapeChanged;  
+    }  
+    public class MyEventArgs : EventArgs   
+    {  
+        // class members  
+    }  
+    public class Shape : IDrawingObject  
+    {  
+        public event EventHandler ShapeChanged;  
+        void ChangeShape()  
         {  
-            // class members  
+            // Do something here before the event…  
+
+            OnShapeChanged(new MyEventArgs(/*arguments*/));  
+
+            // or do something here after the event.   
         }  
-        public class Shape : IDrawingObject  
+        protected virtual void OnShapeChanged(MyEventArgs e)  
         {  
-            public event EventHandler ShapeChanged;  
-            void ChangeShape()  
-            {  
-                // Do something here before the event…  
-  
-                OnShapeChanged(new MyEventArgs(/*arguments*/));  
-  
-                // or do something here after the event.   
-            }  
-            protected virtual void OnShapeChanged(MyEventArgs e)  
-            {  
-                ShapeChanged?.Invoke(this, e);  
-            }  
+            ShapeChanged?.Invoke(this, e);  
         }  
-  
     }  
-    ```  
+
+}  
+```  
   
 ## Example  
 The following example shows how to handle the less-common situation in which your class inherits from two or more interfaces and each interface has an event with the same name. In this situation, you must provide an explicit interface implementation for at least one of the events. When you write an explicit interface implementation for an event, you must also write the `add` and `remove` event accessors. Normally these are provided by the compiler, but in this case the compiler cannot provide them.  
   
 By providing your own accessors, you can specify whether the two events are represented by the same event in your class, or by different events. For example, if the events should be raised at different times according to the interface specifications, you can associate each event with a separate implementation in your class. In the following example, subscribers determine which `OnDraw` event they will receive by casting the shape reference to either an `IShape` or an `IDrawingObject`.  
   
- [!code-csharp[csProgGuideEvents#10](../../../csharp/programming-guide/events/codesnippet/CSharp/how-to-implement-interface-events_1.cs)]  
+ [!code-csharp[csProgGuideEvents#10](../../../csharp/programming-guide/events/codesnippet/CSharp/how-to-implement-interface-events_1.cs#everything)]
```

Comment:
As far as I can tell based on VS Code preview, this way the `#region` lines won't be visible here.